src/components/common/DetailsList.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div id="correction-comment">
```

Comment:
I believe the other components use their name as their ID.

Reply:
Just saw this, fixed.